shm/Cargo.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+#include <string>
+
+#include<string>
```

Comment:
Why include string twice in the same file?

Reply:
+1

Reply:
fixed